### ISSUE TITLE
VR-Caps-Unity-RL

### ISSUE THREAD

**Issue by tr1f0rc3** [CapsuleEndoscope/VirtualCapsuleEndoscopy#4]
I installed VR-Caps-Unity-RL as shown in README.md.
Two errors come out, 

~~~
Rebuilding Library because the metadata folder could not be found!
Assets\Scripts\CoverageMagnetAgent.cs(98,26): error CS0115: 'CoverageMagnetAgent.OnActionReceived(float[])': no suitable method found to override
~~~

Would you help me to solve the issue?
This doesn't occur in the ml-agent example project - ball3d
https://github.com/Unity-Technologies/ml-agents/tree/main/Project/Assets/ML-Agents/Examples/3DBall

Environment: Windows 10; Unity 2020.3.13f1; ML-Agent 18; C#: ASP.NET Core 5.0
Thank you.

**Comment by omercelik-cs** [comment]
If you've installed the ml-agents properly e.g. as described in the readme, it's probably the version mismatch. The version that works fine is the following: 
{
  "name": "com.unity.ml-agents",
  "displayName": "ML Agents",
  "version": "1.1.0-preview",
  "unity": "2018.4",
  "description": "Use state-of-the-art machine learning to create intelligent character behaviors in any Unity environment (games, robotics, film, etc.).",
  "dependencies": {
    "com.unity.barracuda": "1.0.1"
  }
}

**Comment by tr1f0rc3** [comment]
Thank you @omercelik-cs, I installed 2018.4 version and run and It works.

**Comment by tr1f0rc3** [comment]
@omercelik-cs 
Thank you for your reply. I installed the mentioned version of unity and ml-agent. All the compile errors are gone! I found that the error `CoverageMagnetAgent.OnActionReceived(float[])': no suitable method found to override`  occurred since the input parameter type is different from the latest ml-agents.
But there are two runtime errors that occurred:
~~~
Agent component was not found on this gameObject and is required.
UnityEngine.Debug:Assert(Boolean, String)
Unity.MLAgents.DecisionRequester:Awake() (at C:/libs/ml-agents_1.1.0/com.unity.ml-agents/Runtime/DecisionRequester.cs:48)
NullReferenceException: Object reference not set to an instance of an object
MeshMaker.Update () (at Assets/Scripts/MeshMaker.cs:66)
~~~
